moveit_ros/planning/planning_scene_monitor/demos/demo_scene.cpp

```diff
@@ -75,23 +85,28 @@ void sendKnife()
   co.primitive_poses[0].position.z = -0.2;
   co.primitive_poses[0].orientation.w = 1.0;
 
-  pub_aco.publish(aco);
+  pub_aco->publish(aco);
   sleep(1);
-  pub_aco.publish(aco);
-  ROS_INFO("Object published.");
-  ros::Duration(1.5).sleep();
+  pub_aco->publish(aco);
+
+  RCLCPP_INFO(node->get_logger(), "Object published.");
```

Comment:
Add global logger instance

Reply:
This's just a demo, and the logger is used once, should I add a global logger anyway .?

Reply:
Yes please, especially demo code should follow our best practices.